core/src/main/java/com/klaytn/caver/wallet/keyring/KeyringFactoryWrapper.java

```diff
@@ -0,0 +1,82 @@
+package com.klaytn.caver.wallet.keyring;
+
+import org.web3j.crypto.CipherException;
+
+import java.io.IOException;
+import java.util.List;
+
+public class KeyringFactoryWrapper {
```

Comment:
Please add description for javadoc.

Reply:
Ok I will 👍 .
Thanks for your feedback.

Reply:
@jimni1222 
I added descriptions here +_+
[Add descriptions to KeyringFactoryWrapper](https://github.com/klaytn/caver-java/pull/215/commits/6ec62449d882913846c787e54d89c6e2bdb2ee54)